src/python/WMCore/FwkJobReport/XMLParser.py

```diff
@@ -233,7 +233,7 @@ def runHandler():
                           if "ID" in lumi.attrs]
 
                 runInfo = Run(runNumber = runId)
-                runInfo.lumis.extend(lumis)
+                runInfo._lumis.extend(lumis)
```

Comment:
Help me understand this. I would have thought that with the @property decorator would allow you to just lumis?